examples/asr/transcribe_speech.py

```diff
@@ -400,9 +400,16 @@ def autocast(dtype=None):
         logging.info(f"Finished transcribing {len(filepaths)} files !")
     logging.info(f"Writing transcriptions into file: {cfg.output_filename}")
 
-    # if transcriptions form a tuple of (best_hypotheses, all_hypotheses), extract just best hypothesis
+    # if transcriptions form a tuple of (best_hypotheses, all_hypotheses)
     if type(transcriptions) == tuple and len(transcriptions) == 2:
-        transcriptions = transcriptions[0]
+        if isinstance(transcriptions[1][0], list) and isinstance(transcriptions[1][0][0], Hypothesis):
```

Comment:
Triple list shouldn't be needed. Let's stick with two nest list max. It will break multiple other parts external to Nemo

Reply:
removed